mino/minogrpc/mod.go

```diff
@@ -316,3 +335,20 @@ func (m *Minogrpc) listen(socket net.Listener) {
 	// server has well started after that point.
 	<-m.started
 }
+
+func serverTracingDecorator(ctx context.Context, span opentracing.Span, method string,
```

Comment:
I would comment what this function is used for.

Reply:
Also a function's name should preferable describe an action, with a verb. Maybe `decorateServerTrace` ?